apps/src/util/loadVideos.js

```diff
@@ -24,18 +24,14 @@ export function loadVideos(force_fallback) {
 // Parameter player is either "youtube" or "fallback".
 function setupVideos(player) {
   if (player === 'fallback') {
-    const doc = document,
-      video_css = doc.createElement('link'),
-      video = doc.createElement('script'),
-      video_ie = doc.createElement('script');
+    const doc = document;
+    let video_css = doc.createElement('link');
+    let video = doc.createElement('script');
 
     video_css.type = 'text/css';
     video_css.href = '/css/video-js.min.css';
     video_css.rel = 'stylesheet';
     doc.getElementsByTagName('head')[0].appendChild(video_css);
-    video_ie.type = 'text/javascript';
-    video_ie.src = '/js/videojs-ie8.min.js';
```

Comment:
I'm confused, does this mean we don't support IE8 anymore?

Reply:
Yes, as noted in the linked PR, we won't support IE below 11 with the updated fallback player.

Reply:
Ah, there's the problem I'm reviewing out of order. Sorry - thanks!